code-examples/promisify.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
+
+// Promisified utility
+const promisifiedReadFile = (file, encoding)  => {
+  return new Promise((resolve, reject) => {
```

Comment:
Do you think is necessary to disclose what is a promisify method instead of just using `require('util').promisify`?

Reply:
I'm actually not going to include this on the talk, and yeah, I would use promisify if I was creating some code in production, however, I want to demonstrate the *bridge* between *callbacked* code, and promises, so here it would be something like a demo of "how to convert code that runs with callbacks to code that now returns promises"